src/main/java/hw32/HibernateSession.java

```diff
@@ -0,0 +1,32 @@
+package hw32;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.service.ServiceRegistry;
+
+public class HibernateSession {
+
```

Comment:
It's better to nami it like HibernateSessionConfig or smth to not override existed class